packages/components/src/core/ContentCard/index.tsx

```diff
@@ -133,9 +139,13 @@ const ContentCard = React.forwardRef<HTMLDivElement, ContentCardProps>(
           imageSize={imageSize}
           image={image}
           imagePosition={imagePosition}
+          className={classes?.cardMedia}
```

Comment:
nit: instead of having optional chaining throughout, should we just:

```tsx
{ classes = EMPTY_OBJECT } = props

const { cardMedia, cardContent, ... } = classes;
```

Reply:
Oh, sure! That’s actually better. I’ll fix them ASAP.